app/assets/src/components/BulkUploadImport.jsx

```diff
@@ -90,6 +92,8 @@ class BulkUploadImport extends React.Component {
     );
     this.initializeSelectTag();
     this.initializeSelectAll();
+
+    if (this.state.imported) this.resolveSampleNames();
```

Comment:
Do we need to call this here? We already call it at line 353, which is where `imported = true` gets set.

I worry that it'll make all these api calls every time the component re-renders (which can be often)

Reply:
you're right it wasn't needed 👍